CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributor Guide
+
+## Environment setup and workflow
+- Clone the repo
+- Create a fresh python environemt with `.binder/requirements.txt` file
+- Open `jupyter lab` or `jupyter notebook` (so that jupytext extension can take effect)
+- Then right-click on a MyST markdown notebook, choose `Open With` -> `Notebook`
+or `Jupytext Notebook` (more info [here](https://nasa-navo.github.io/navo-workshop/00_SETUP.html#handling-notebooks-in-myst-markdown-format)).
```

Comment:
we should link more upstream than NAVO, e.g. numpy

Reply:
Can you share the link? Last I checked numpy docs, I got confused because they were instructing to "pairing the notebook and md" but then I learned from you that that's not required.

Reply:
I knew it would bite me back, we need to cleanup the numpy docs first, 😅 : https://github.com/numpy/numpy-tutorials/issues/186